db/seeds.rb

```diff
@@ -7,4 +7,4 @@
 #   Mayor.create(name: 'Emanuel', city: cities.first)
 #
 # Remember to ensure that the commands in this file are idempotent!
-Instance.find_or_create_by(name: 'Default', host: '*')
+Instance.find_or_create_by(name: 'Default', host: 'coursemology.org')
```

Comment:
...you just changed the meaning of my default instance.

what if a person sets up his own Coursemology instance, not at coursemology.org?